presto-product-tests/conf/docker/common/jdbc_db.yml

```diff
@@ -24,3 +24,12 @@ services:
       MYSQL_PASSWORD: swarm
       MYSQL_ROOT_PASSWORD: swarm
       MYSQL_DATABASE: test
+
+  sqlserver:
+    hostname: sqlserver
+    image: 'microsoft/mssql-server-linux'
+    ports:
+      - '1433:1433'
```

Comment:
Unlike MySQL and PostgreSQL, we can't pass port as the parameter. We will have to create a custom docker image on top of sql-server. As discussed, for now I'll just run on 1433